log_analyzer/packet.py

```diff
@@ -26,4 +33,4 @@ def __str__(self):
         return "Receiver: {}\nSender: {}\nEvents: {}".format(self.get_receiver(), self.get_sender(), self.get_events())
 
     def __repr__(self):
-        return "\n" + str(self) + "\n"
\ No newline at end of file
+        return "\n" + str(self) + "\n"
```

Comment:
SOmetimes, there is a last line newline, sometimes not. We can think on using pep8 and possibly pylint on the Python code. Maybe pep8 also as part of the CI loop.

Reply:
I will make the pep_8 test and make sure the convention is everywhere inside the log_analyzer